src/components/signup/Signin.jsx

```diff
@@ -1,3 +1,92 @@
-export function Signin() {
-  return <></>;
+import { useState } from "react";
+import "./signup.css";
+export function Signin({ setIsSignUp }) {
+  const [showPassword, setShowPassword] = useState(false);
+  const [loginForm, setLoginForm] = useState({
+    email: "",
+    password: "",
+  });
+
+  const changeHandler = event => {
+    event.preventDefault();
+    setLoginForm(() => ({
+      ...loginForm,
+      [event.target.name]: event.target.value,
+    }));
+  };
+
+  const toggleShowPassword = e => {
+    e.preventDefault();
+    setShowPassword(() => (showPassword ? false : true));
+  };
+
+  const handleLoginSubmit = e => {
+    e.preventDefault();
+  };
```

Comment:
Is a preventDefault needed at all function calls?  I believe it's only necessary in the submit function being called. Do have a look into it.

Reply:
Since I have 2 forms, one each for signup & login, I am using preventDefault in both the form submissions.